uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/leave-request-popup-controller.js

```diff
@@ -64,13 +65,17 @@ define([
         }
       };
       vm.uiOptions = {
-        isAdmin: false,
         isChangeExpanded: false,
+        isManager: false, //when the dialog is opened by manager or admin
+        isOwner: false, //when the dialog is opened by the owner
```

Comment:
I like the idea but here I think number of comparisons will still remain same, moreever a boolean is better and smaller comparison than string